woof-code/rootfs-skeleton/usr/sbin/sfs_load

```diff
@@ -74,15 +80,14 @@ if [ -L /bin -o -L /sbin -o -L /lib -o -L /lib64 -o -L /lib32 -o -L /libx32 ]; t
     rm -rf /mnt/$MNTFLD
        
     if [ "$FLD_INVALID" != "" ]; then
-      /usr/lib/gtkdialog/box_ok "sfs_load" error "$(gettext "This sfs should not be loaded because it has the wrong file structure")"
-      exit 1
+      /usr/lib/gtkdialog/box_yesno "sfs_load" warning "$(gettext "The directory $FLD_INVALID in this sfs will overwrite the system symlink. This may cause problems. Do you want to continue?")"
+      [ $? == 1 ] && exit 1
```

Comment:
In which case would you think a user would like to overwrite with links and break the system?

Reply:
That's a good question, #3027 was meant to protect users. If this SFS is dangerous, there's no reason to allow loading. If it's not dangerous, there's no reason to do this check.